cypress/integration/licenses.js

```diff
@@ -1,7 +1,7 @@
 describe('Licenses', () => {
     it('Licenses loaded', () => {
-        cy.get('[data-attr=menu-item-settings]').click()
-        cy.get('[data-attr=menu-item-instance-licenses]').click()
+        cy.get('.whoami').click() // Top navigation dropdown
```

Comment:
Note: Using class names in integration tests is mixing concerns. E.g. here you need to add a comment for the code to be understandable.

Why not add a data-attr?